src/vttLib/transfer.py

```diff
@@ -60,16 +62,23 @@ def merge_from_file(font: fontTools.ttLib.TTFont, path: os.PathLike) -> None:
         raise vttLib.VTTLibArgumentError("'maxp' table not found in target font.")
 
     TABLES_TO_MERGE = ("TSI0", "TSI1", "TSI2", "TSI3", "TSI5")
-    TABLES_TO_MERGE_OPTIONAL = ("TSIC", "cvt ", "cvar")
+    tables_to_merge_optional = ["TSIC", "cvt "]
```

Comment:
I think `cvt ` is only needed for `cvar`, because fT adjusts some `cvar` stuff while looking at `cvt `, so try moving `"cvt "` down into the `keep_cvar` guard.

Reply:
cvt is still required because all the glyph programs in the glyf table reference it. It's also used when compiling TSIC as the source from which the deltas are made (__init__.py:728-736,746). (cvt is to cvar what glyph outlines are to gvar). There is no other source for this data, except the Control Program from TSI3, but that gets into the realm of being a high-level language compiler instead of just an assembler.

Reply:
Wait, don't we compile a fresh cvt from TSIwhatever? 🤔 I forgot about that part of the code base.

Reply:
I hadn't noticed this code. I've made the change and updated vttLib_test.test_TSIC_compile() to also verify the cvt table against the VTT version. 

This makes the inclusion of the cvt table governed by --keep-cvar. We could similarly add a --keep-cvt to both merge and compile. I'm fine leaving it as is for now. Not sure why I'd want one but not the other...

Reply:
Thanks! Let's just do the existing cvar switch, cvt was only roped in because of a technicality.